frontend/ui/elements/mass_storage.lua

```diff
@@ -54,15 +54,19 @@ function MassStorage:start(never_ask)
 
     if not never_ask and self:requireConfirmation() then
         local ConfirmBox = require("ui/widget/confirmbox")
-        UIManager:show(ConfirmBox:new{
+        UIManager:show(ConfirmBox:new {
```

Comment:
No need to change ;-)
```suggestion
        UIManager:show(ConfirmBox:new{
```

Reply:
I now see it's my IDE that does this! I wasn't aware of this change. I'll fix it.

Reply:
No need anymore to change it, because you already merged my commit.

Reply:
There would still be a need of sorts to change it if I'd accidentally put it in that way but for something this minor it hardly matters who commits it or how. ^_^ (I hope you agree, anyway!)